dataprofiler/profilers/utils.py

```diff
@@ -686,3 +686,21 @@ def perform_chi_squared_test_for_homogeneity(
     results["p-value"] = p_value
 
     return results
+
+
+def merge_list_of_profiles(list_of_profiles):
+    """Using the profile object's `__add__` method, to merge all the
+        provided profiles in the list into a single profile, which
+        is returned by the function.
+
+    :param list_of_profiles: List of Data Profiler profile objects.
+        Acceptable profile types are `StructuredProfiler` and
+        `UnstructuredProfiler`.
```

Comment:
technically, this could be anything with an `add`? or might we have to curate specific code for this top level?

Reply:
True it could be anything with an `__add__`. That's completely fair. Making it more reusable at any level is 🥇